appveyor.yml

```diff
@@ -11,13 +11,19 @@ before_build:
 
 platform: Any CPU
 
+environment:
+  _CL_: /c /analyze /analyze:quiet- /W4 /analyze:autolog:ext prefast.sarif
+
 configuration:
 #  - Debug
   - Release
 
 build:
   project: src/Everything.sln
-  
+
+before_test:
```

Comment:
>before_test [](start = 0, length = 11)

Would after_build be more appropriate than before_test? I assume that after_build runs even when the build fails. But before_test probably only runs when the build succeeds.